howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+"""
+This module contains example snippets on how to request cached data from a direct receiver. These examples assume
+that the direct receiver has already been started, and that it was created from a messaging service which has
+already been connected.
+"""
+
+from solace.messaging.utils.correlation_completion_listener import CorrelationCompletionListener
+from solace.messaging.utils.cache_request_outcome import CacheRequestOutcome
+from solace.messaging.resources.topic_subscription import TopicSubscription
+from solace.messaging.resources.cached_message_subscription_request import CachedMessageSubscriptionRequest
+
+class MyCorrelationCompletionListener(CorrelationCompletionListener):
+    def on_completion(result: CacheRequestOutcome, correlation_id: int, e: Exception):
+        print("Completed!")
+
+class HowToUseCachedMessageSubscriptionRequests:
+    """
+    Sampler for making cache requests. Usage of cache subscriptions require
+    configuration and hosting of SolCache application.
+    """
+    def create_subscription_request_to_receive_cached_and_live_messages(subscription_expression: str,
+                                                                        cache_name: str,
+                                                                        cache_access_timeout: int,
+                                                                        direct_message_receiver: 'DirectMessageReceiver'):
+        """
+        An example how to create and perform CachedMessageSubscriptionRequest to receive a
+        mix of live and cached messages matching specified topic subscription.
+        Args:
+            subscription_expression(str): topic subscription expression
+            cache_name(str): name of the solace cache to retrieve from
+            cache_access_timeout(int): solace cache request timeout, in milliseconds
```

Comment:
Since there is no default value for this timeout we should be giving guidance on what's reasonable here.  PubSub+ Cache and CCSMP default is 10 seconds (10000ms)
This applies to all the functions that use this timeout

Reply:
@TamimiGitHub, the valid range and default value for the parameter is documented in both the API reference doc and the API dev guide. It seems like feature creep to further document it here. However, I'm unclear on to what extent the HowTos should inform an app dev on best practices, and if including guidance on parameter values should be done in the HowTos?